spec/parameters/query/embedded.yml

```diff
@@ -3,6 +3,10 @@ in: query
 description: |
   When true, the endpoint also returns all the embedded aggregate transactions.
   Otherwise, only top-level transactions used to calculate the block transactionsHash are returned.
+  **Note:** This field does not work when combined with the ``address`` parameter. This is,
```

Comment:
Not related to this PR but do you know what this means above? - 
``Otherwise, only top-level transactions used to calculate the block transactionsHash are returned``

Reply:
No, I don't, and the OpenAPI docs are full of this cryptic stuff. My plan is to burn all OpenAPI docs to the ground and write them all again.
My wish might be granted, since Kristy wants to migrate to GraphQL...